### PR TITLE
Remove all arabic dead links an move working links to ae.m3u

### PR DESCRIPTION
**Pull request by Jariix**
Dead Links:

http://95.170.215.107/hls/KSA-Sport2-Bakup-M.m3u8	
http://95.170.215.107:80/hls/m3u8/ON-Sport.m3u8	
http://95.170.215.107/hls/Sky-New-A-Backup-M.m3u8	
http://95.170.215.107:80/hls/BBCArabic-Backup-M.m3u8	
http://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/chunklist_b1264000.m3u8?v&__mn_sid=17174366775755575437	
https://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/chunklist_b1264000.m3u8	
http://qtv-i.akamaihd.net/hls/live/213233/QTV/s_500330.m3u8	
https://stream.skynewsarabia.com/hls/sna_720.m3u8	
http://live.hibridcdn.net/cnbc/liveabr/cnbc/live1/chunks.m3u8?gmswf=2756198?3292634087	
http://admdn2.cdn.mangomolo.com/nagtv/smil:nagtv.stream.smil/chunklist_b4000000_t64MTA4MHA=.m3u8	
https://stream-05.dc3.dailymotion.com/sec(86G48EQSWKUUFPhHXEV5xihAZ6OwwXGV3CGnvlVY4CA)/dm/3/x74wje5/s/live-3.m3u8	
http://live.hibridcdn.net/cnbc/liveabr/cnbc/live3/chunks.m3u8?gmswf=15392984?787764643	
http://95.170.215.107/hls/m3u8/KSA-Sport1-Bakup.m3u8	
https://bracnet.iptv.jadoodigital.com/live/al_jazeera/playlist.m3u8	
http://103.205.133.42:1935/local/al_arabia_720/index.m3u8

### CODE REVIEW

**Comment by 0x41mmarVM** [comment]
- I notice that a number of these are duplicates. For example you (rightly) removed two links for Al Hiwar

`
http://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/chunklist_b1264000.m3u8?v&__mn_sid=17174366775755575437
`
`
https://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/chunklist_b1264000.m3u8
`
But I notice Al Hiwar is also available in uk.m3u with a working link. Not sure what the implication here is, just something I noticed.

- There is also an interesting pattern where some of these are easily fixed by replacing a single quality link with the master playlist.

Example: 
`http://admdn2.cdn.mangomolo.com/nagtv/smil:nagtv.stream.smil/chunklist_b4000000_t64MTA4MHA=.m3u8`
is not working because it's specific for the 4Mb bitrate. Replacing the last part with playlist.m3u8 fixes it:
`http://admdn2.cdn.mangomolo.com/nagtv/smil:nagtv.stream.smil/playlist.m3u8`

- One of the ones you removed seems to be working fine
`https://stream.skynewsarabia.com/hls/sna_720.m3u8`
but it should be changed anyway, the correct (master) link is 
`https://stream.skynewsarabia.com/hls/sna.m3u8`

**Comment by 0x41mmarVM** [comment]
Most of these are not Emirati channels, they don't belong in ae.m3u

**Comment by Jariix** [comment]
Where is the Arabic m3u? I want to sort by language.

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -333,72 +333,8 @@ https://aldirect.rtmp.huya.com/huyalive/1099531752610-1099531752610-864355038674
 https://aldirect.rtmp.huya.com/huyalive/1099531752611-1099531752611-86435508162396160-2199063628678-10057-A-0-1.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="",学而思 Ⅰ 高中三年级
 https://aldirect.rtmp.huya.com/huyalive/1099531752612-1099531752612-86435512457363456-2199063628680-10057-A-0-1.m3u8
-#EXTINF:-1 tvg-id="ext" tvg-name="" tvg-language="" tvg-logo="" group-title="Sport",KSA SPORTS 2 HD
-http://95.170.215.107/hls/KSA-Sport2-Bakup-M.m3u8
-#EXTINF:-1 tvg-id="ext" tvg-name="" tvg-language="" tvg-logo="" group-title="Sport",KSA SPORTS 2 SD
-https://edge-1192-ch-gv.filmon.com/live/1778.high.stream/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:339]: Link works.

File: channels/unsorted.m3u [hunk 2]
```diff
@@ -333,72 +333,8 @@ https://aldirect.rtmp.huya.com/huyalive/1099531752610-1099531752610-864355038674
...
-#EXTINF:-1 tvg-id="ext" tvg-name="" tvg-language="" tvg-logo="" group-title="Sport",KSA SPORTS 2 low
-https://edge-1192-ch-gv.filmon.com/live/1778.low.stream/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:341]: Link works.

File: channels/unsorted.m3u [hunk 3]
```diff
@@ -333,72 +333,8 @@ https://aldirect.rtmp.huya.com/huyalive/1099531752610-1099531752610-864355038674
...
-#EXTINF:-1 tvg-id="ADSports1.net" tvg-name="" tvg-language="" tvg-logo="" group-title="Sport",Abu Dhabi Sports 1
-https://admdn1.cdn.mangomolo.com/adsports1/smil:adsports1.stream.smil/chunklist_b1800000_t64NzIwcA==.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:343]: Link works.

File: channels/unsorted.m3u [hunk 4]
```diff
@@ -333,72 +333,8 @@ https://aldirect.rtmp.huya.com/huyalive/1099531752610-1099531752610-864355038674
...
-#EXTINF:-1 tvg-id="ADSports2.net" tvg-name="" tvg-language="" tvg-logo="" group-title="Sport",Abu Dhabi Sports 2
-https://admdn5.cdn.mangomolo.com/adsports2/smil:adsports2.stream.smil/chunklist_b1800000_t64NzIwcA==.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:345]: Link works.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again and return the working links back. Also resolve the conflict in the file so we can approve this pull request.

**Comment by 0x41mmarVM** [comment]
@Jariix Sorting by language doesn't require you to move the files, just to define the language in the line. You only need to move them to sort by country. Check the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md), and the Readme.

File: channels/ae.m3u [hunk 5]
```diff
@@ -17,32 +31,203 @@ https://admdn2.cdn.mangomolo.com/adtv/smil:adtv.stream.smil/chunklist.m3u8
...
+http://dmitwlvvll.cdn.mangomolo.com/dubaisportshd/smil:dubaisportshd.smil/chunklist_b1600000.m3u8?2957056943
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",AlWousta TV
+https://svs.itworkscdn.net/alwoustalive/alwoustatv.smil/playlist_1080p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",AlRai TV
+https://stream02.fasttelco.net/live/alrai.stream/chunklist_w498917617.m3u8?v=2957056943
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Alkuwait Plus Live
+https://kwmedia.hibridmedia.com/kwmedia/kwtvsports_dabr/kwmedia/kwtvsports_1080p/chunks.m3u8?v=2957056943
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Alkuwait Plus Movies
+https://kwmedia.hibridmedia.com/kwmedia/kwtvkids_dabr/kwmedia/kwtvkids_1080p/chunks.m3u8?v=2957056943
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Alkuwait Aloula
```
Comment (Link4K) [inline channels/ae.m3u:85]: Already in kw.m3u8

File: channels/ae.m3u [hunk 6]
```diff
@@ -17,32 +31,203 @@ https://admdn2.cdn.mangomolo.com/adtv/smil:adtv.stream.smil/chunklist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Bahrain Sports
+http://185.105.4.106:1935/live/Bahrain%20Sports/chunklist.m3u8?v=3116400149
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Bahrain Sports 2
+http://185.105.4.107:1935/live/Bahrain%20Sports%202/chunklist.m3u8?v=2957056943
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Canadian Arab TV (Canada)
+http://192.226.228.32:8080/hls/catv/index.m3u8
 #EXTINF:-1 tvg-id="Citruss TV ARB" tvg-name="Citruss TV ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6lTEvrU.jpg" group-title="",Citruss TV
 https://citrusstv.akamaized.net/hls/live/687285/CTV/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="",Citruss TV HD
+https://citrusstv.akamaized.net/hls/live/687285/CTV/01.m3u8
```
Comment (Link4K) [inline channels/ae.m3u:144]: Duplicate. Above stream is adaptive

**Comment by Link4K** [comment]
Most channels are not from the country ae

Please move each channel to the correct country file and avoid duplicated streams.

File: channels/ae.m3u [hunk 7]
```diff
@@ -61,21 +246,69 @@ https://shls-mbcbollywood-prod.shahid.net/mbcbollywood-prod.m3u8
...
+http://partwo.cdn.mangomolo.com/omlive/smil:omlive.stream.smil/chunklist_b4500000_t64MTA4MHA=.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Oman Althakafia
+http://partwo.cdn.mangomolo.com/omcultural/smil:omcultural.stream.smil/chunklist_b4500000_t64MTA4MHA=.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Oman Sports
+http://partne.cdn.mangomolo.com/omsport/smil:omsport.stream.smil/chunklist_b4500000_t64MTA4MHA=.m3u8
+#EXTINF:-1 tvg-id="ext" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Osn Beity
+http://82.212.74.99:8000/live/hls/8164.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/ibUfEGr.png" group-title="",Peace TV
 http://peacetv.api.visionip.tv/live/ASHTTP/peacetv-peacetv-peacetv-english-hsslive-25f-16x9-MB/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",RT Arabic HD
```
Comment (Link4K) [inline channels/ae.m3u:269]: In ru.m3u

File: channels/ae.m3u [hunk 8]
```diff
@@ -61,21 +246,69 @@ https://shls-mbcbollywood-prod.shahid.net/mbcbollywood-prod.m3u8
...
 https://svs.itworkscdn.net/kablatvlive/kabtv1.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Soumalian TV
+http://cdn.mediavisionuae.com:1935/live/stv.stream/chunklist_w1827589767.m3u8?v=2957056943
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Spacetoon
+http://stream.spacetoon.com:1935/live/stchannel/smil:livesmil.smil/chunklist_w693439211_b346000_slAR.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Tamazight
+https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/tamazight_tv8_snrt/hls_snrt/tamazight_tv8_snrt-avc1_1500000=4-mp4a_130400_qad=1.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/nsxrqta.jpg" group-title="",Toheed TV
 http://51.158.73.206/hls/stream.m3u8
+#EXTINF:-1 tvg-id="TRTAlArabiya.tr" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",TRT Arabic
```
Comment (Link4K) [inline channels/ae.m3u:307]: In the Turkey playlist

**Review by Link4K — CHANGES_REQUESTED** [review]

**Comment by Jariix** [comment]
Hello, can you delete this pull request now and I will update the unsorted.m3u in the these days. Thank you in advance.

**Pull request closed without merging**